core/new-gui/src/app/workspace/service/operator-metadata/operator-metadata.service.ts

```diff
@@ -13,6 +13,10 @@ export const EMPTY_OPERATOR_METADATA: OperatorMetadata = {
   groups: []
 };
 
+const addDictionaryAPIAddress = '/api/resources/dictionary/';
+const getDictionaryAPIAddress = '/api/upload/dictionary/';
```

Comment:
Define these constant address as 
`export const ADD_DICTIONARY_API_ADDRESS = '/api/resources/dictionary/';` 
to be consistent with the address defined previously. 

For instance: 
`export const MOCK_OPERATOR_METADATA_ENDPOINT = 'resources/operator-metadata';`

Reply:
Good catch! I don't know why the operator-metadata is varied but I'll fix these for this and the rest of  PRs